src/index.js

```diff
@@ -389,6 +393,7 @@ function appendChildOrInsertBefore (host, newNode, refNode) {
 
   if (nodeType === 'node') {
     if (canPatchNativeAccessors) {
+      nodeToParentNodeMap.set(newNode, host);
```

Comment:
For some reason this was failing in Opera but not other browsers. We really should have been doing this anyways to update the parent node mapping since it's proxying off to a native method.